tests/integration/TEC/Tickets/QR/QR_Test.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace TEC\Tickets\QR;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use tad\WP\Snapshots\WPHtmlOutputDriver;
+
+/**
+ * Class QR_Test.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\QR
+ */
+class QR_Test extends \Codeception\TestCase\WPTestCase {
+	use MatchesSnapshots;
```

Comment:
I would use the `codeception-snapshot-asssertion` library: it's made for Codeception, uses Codeception-format snapshots, and produces readable snapshots.
This snapshot library was made for PHPUnit and jerry-rigged into Codeception when there was no alternative.